web-components/src/components/input-file/InputFile.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import reset from "@/wc_scss/reset.scss";
+import { customElement, html, LitElement, query, property, internalProperty } from "lit-element";
+import styles from "./scss/module.scss";
+import "@/components/button/Button";
+
+import { nothing } from "lit-html";
+@customElement("md-input-file")
```

Comment:
maybe call it more like a "file upload" ?

Reply:
Not sure. HTML Element calls as "input type file". What do you think?